airspeed/__init__.py

```diff
@@ -502,7 +519,7 @@ def parse(self):
 class DictionaryLiteral(_Element):
     START = re.compile(r'{[ \t]*(.*)$', re.S)
     END = re.compile(r'[ \t]*}(.*)$', re.S)
-    KEYVALSEP = re.compile(r'[ \t]*:[[ \t]*(.*)$', re.S)
+    KEYVALSEP = re.compile(r'[ \t]*:[ \t]*(.*)$', re.S)
```

Comment:
This appears to have been a regex syntax error (superfluous bracket "["). The messed up code highlighting of my editor made me aware of this. :)